src/games/gameitem.py

```diff
@@ -218,99 +218,26 @@ def tooltip(self, data):
-                 "{player}"
-                 "</td>")
-        order = [pname, country] if align == 'right' else [country, pname]
-        player_row = "<tr>{}{}</tr>".format(*order)
-
-        if isinstance(player, str):
-            return player_row.format(alignment=align, player=player)
-        else:
-            return player_row.format(
-                country_icon=self._country_icon_fmt(player),
-                alignment=align,
-                player=self._player_fmt(player))
-
-    def _country_icon_fmt(self, player):
-        icon_path_fmt = os.path.join("chat", "countries", "{}.png")
-        icon_path = icon_path_fmt.format(player.country.lower())
-        icon_abs_path = os.path.join(util.COMMON_DIR, icon_path)
-        return "<img src='{}'>".format(icon_abs_path)
-
-    def _player_fmt(self, player):
-        if player == self._me.player:
-            pformat = "<b><i>{}</b></i>"
-        else:
-            pformat = "{}"
-        player_string = pformat.format(player.login)
-        if player.rating_deviation < 200:   # FIXME: magic number
-            player_string += " ({})".format(player.rating_estimate())
-        return player_string
-
-    def _observers_tooltip(self, observers):
-        if not observers:
-            return ""
-
-        observer_fmt = "{country_icon} {observer}"
-
-        observer_strings = [observer_fmt.format(
-            country_icon=self._country_icon_fmt(observer),
-            observer=observer.login)
-            for observer in observers]
-        return "Observers: " + ", ".join(observer_strings)
-
-    def _mods_tooltip(self, mods):
-        if not mods:
-            return ""
-        return "<br/>With: " + "<br/>".join(mods.values())
 
     def format(self, teams, observers, mods):
-        teamtip = self._teams_tooltip(teams)
-        obstip = self._observers_tooltip(observers)
-        modtip = self._mods_tooltip(mods)
-
-        if modtip:
-            modtip = "<br/>" + modtip
-
-        return self.TIP_FORMAT.format(teams=teamtip,
-                                      observers=obstip,
-                                      mods=modtip)
+        def transform_playercountry_to_path(player):
+            if player is None or isinstance(player, str):
+                return
+            else:
+                icon_path_fmt = os.path.join("chat", "countries", "{}.png")
+                icon_path = icon_path_fmt.format(player.country.lower())
+                icon_abs_path = os.path.join(util.COMMON_DIR, icon_path)
+                player.countrypath = icon_abs_path
+        for team in teams:
+            for player in team:
+                transform_playercountry_to_path(player)
+        for observer in observers:
+            transform_playercountry_to_path(observer)
+
+        template_abs_path = os.path.join(util.COMMON_DIR, "games", "gameitem.css")
+        with open(template_abs_path,"r") as templatefile:
+            template = jinja2.Template(templatefile.read())
```

Comment:
I suggest making this template once, in __init__. No point in reading and compiling the template from scratch every time.